src/clickmap_nav/tool_retrieval_interface.py

```diff
@@ -37,6 +48,12 @@ def __init__(self, robot, upload_path, model_path, silhouette=None, silhouetteAc
             'right_fisheye_image',
         ] # See conq/cameras_utils.py RGB_SOURCES
 
+        # Get locations to feed to open_ai_nav_client
+        self.waypoint_locations = [name for name in self.name_to_id.keys() if 'waypoint' not in name]

```

Comment:
This is a sincere question that I want your input on. Correct me if I'm wrong, but `name` here refers to the semantic description/label that was given to the waypoint, correct? Do you think `label` would be a more descriptive name for this throughout the code? Or do you prefer `name`?

For example, the highlighted line of code would look like:
```python
self.waypoint_labels = [label for label in self.label_to_id.keys() if 'waypoint' not in label]
```

Typing it out, I understand why you went with `name`, so it's your call on keeping `name` or changing to `label`. 

Regardless, does changing `self.waypoint_locations` to `self.waypoint_names` or `self.waypoint_labels` work for you?

Reply:
Good call, I renamed it to `waypoint_names`. When you make a waypoint, name is ONLY a custom one if given, otherwise the name is set to the waypoint id. Done in [ef9395f](https://github.com/UM-ARM-Lab/conq_python/pull/52/commits/ef9395f3eb57a35291a3f7030eaf2e17f5e23773)